### PR TITLE
sbert embedding

### PR DESCRIPTION
**Pull request by ermalrrapaj**
Created the sbert embedding class and tested that it works with the current setup. The user needs to rebuild the database for the embedder to work.

### CODE REVIEW

File: lmntfy/models/embedding/sbert_embedding.py
Comment (nestordemeure) [inline lmntfy/models/embedding/sbert_embedding.py:1]: Is this the tokeniser used by this embedding? In doubt we could use `None` for now if we don't know.

File: lmntfy.py [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
-# PARAMETERS
 
-data_folder = './data'
-docs_folder = data_folder + '/docs'
-database_path_prefix = data_folder + '/database'
-rebuild_database = False
-use_test_questions = False
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--data",type=str,default="/Users/ermal/workspace/NERSC/NERSC_chatbot/letMeNERSCthatForYou")
```
Comment (nestordemeure) [inline lmntfy.py]: Could we use a relative path here to avoid having a particular username in the default?

File: lmntfy.py [hunk 2]
```diff
@@ -1,40 +1,49 @@
+import os
+os.environ["TOKENIZERS_PARALLELISM"]="False"
```
Comment (nestordemeure) [inline lmntfy.py:2]: why is this needed?

**Review by nestordemeure — CHANGES_REQUESTED** [review]

File: lmntfy.py (hunk 2 above)
Comment (ermalrrapaj) [reply lmntfy.py:2]: When I use the SBERT embedder, if I do not set that environment variable to false I get the following warning:
 
"
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
"
When I tried setting it  to True, the chatbot got stuck at answering the question, so setting it to false seems to both work and remove the warning. It seems that parallelism will lead to deadlocks.

File: lmntfy/models/embedding/sbert_embedding.py
Comment (ermalrrapaj) [reply lmntfy/models/embedding/sbert_embedding.py:1]: I set it to None in the latest push

File: lmntfy.py (hunk 1 above)
Comment (ermalrrapaj) [reply lmntfy.py]: In the latest push, the absolute path to lmntf.py is automatically obtained and used to get the database location. No data argument is needed.
